.github/workflows/e2e_tests.yml

```
@@ -59,84 +55,52 @@ jobs:
+          deployment-target: liam-app
+          environment: ${{ github.event.deployment.environment }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          should-run: ${{ steps.check.outputs.should_run }}
+          ref: ${{ github.event.deployment.sha }}
+          test-project: liam-app
+          url: ${{ github.event.deployment_status.target_url }}
 
-      - name: Post E2E status manually (with gh)
-        if: steps.check.outputs.should_run == 'true' && always()
+      - name: liam-erd-sample-e2e_tests
+        id: check
```

actionlint (1.7.7)

step ID "check" duplicates. previously defined at line:28,col:13. step ID must be unique within a job. note that step ID is case insensitive

(id)